l1-contracts/script/UpgradeMainnetUniFiAVS.s.sol

```diff
@@ -45,7 +45,7 @@ contract UpgradeMainnetUniFiAVS is BaseScript {
 
         bytes memory calldatas;
         bytes4[] memory daoSelectors = new bytes4[](1);
-        daoSelectors[0] = UniFiAVSManager.updateAVSMetadataURI.selector;
+        daoSelectors[0] = UniFiAVSManager.setAllowlistRestakingStrategy.selector;
```

Comment:
This would make `updateAVSMetadataURI` restricted to timelock. 
Did this script already run on mainnet, and is the `updateAVSMetadataURI` already assigned to DAO?